docs/dev/release.md

```diff
@@ -89,8 +89,11 @@ The Java projects must follow a strict sequence, following its dependency hierar
    * [`pass-acceptance-testing`](https://github.com/eclipse-pass/pass-acceptance-testing)
 4. [`pass-docker`](https://github.com/eclipse-pass/pass-docker)
 
+These projects have GitHub workflow automations in place to perform releases that need to be triggered manually. See more detailed [release steps with automations](./release-steps-with-automations.md)
```

Comment:
I prefer absolute paths so `[release steps with autmations](/docs/dev/release-steps-with-automations.md)` as it makes it easier to move files already and globally "search and replace" references to moved files.